lib/vapi_cli/trained-models.py

```diff
@@ -182,6 +182,7 @@ def show(params):
   trained-models deploy (--modelid=<model-id> | --id=<model-id>) [--name=<name>] [--accel=<accel_type>]
                         [--userdnnid=<user_dnn_UUID]
                         [--dnnscriptid=<dnn_script_UUID]
+                        [--datasetid=<dataset_UUID]
```

Comment:
Not sure that I like `--datasetid`; it describes what the value is, not what it is for.
Can we do something like `--savetodataset` or `--savedataset`?